lib/recipes-data/src/lib/takedown.ts

```diff
@@ -3,12 +3,11 @@ import {recipesforArticle, removeAllRecipeIndexEntriesForArticle, removeRecipe}
 import type { RecipeIndexEntry } from './models';
 import {removeRecipeContent} from "./s3";
 
-async function takeRecipeDown(client: DynamoDBClient, canonicalArticleId: string, recipe: RecipeIndexEntry, removeFromDatabase: boolean):Promise<void>
+async function takeRecipeDown(client: DynamoDBClient, canonicalArticleId: string, recipe: RecipeIndexEntry, unconditional: boolean):Promise<void>
```

Comment:
Can `DynamoDBClient` be relocated to the `./dynamo` import, so there's no need to pass it through these functions?